### ISSUE TITLE
'High Contrast Black' theme Token states WCAG AAA contrast compliance

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1020]
**Describe the bug**
The Token components selected and hover state colors contrast ratio is not 7:1 or above.

> What version of the Fundamental Library Styles are you using?
v0.9.1-rc.0

**To Reproduce**
1. Go to https://deploy-preview-928--fundamental-styles.netlify.app/components/token.html
2. Check the Token components selected and hover states' color contrast 

**Expected behavior**
If we want WCAG AAA compliance, this contrast ratio should be 7:1 or above.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81760617-90896380-947c-11ea-99d5-3669a1d0a606.png)
![image](https://user-images.githubusercontent.com/43764832/81756110-a93f4c80-946f-11ea-96b9-778183110978.png)
![image](https://user-images.githubusercontent.com/43764832/81753968-da1c8300-9469-11ea-8acc-3bd4f2de9a25.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by Mike-Diaz** [comment]
This UI element has a white (#FFF) border which is then considered the background color. Comparing #FFF border against #7A5100 results in 7.0:1 contrast ratio and #FFF border against #0F5D94 results in 7.0:1 contrast ratio which pass WCAG AAA contrast ratio requirements.

**Comment by prsdthkr** [comment]
Though the token has a white border the text within (i.e. fd-token__text) does not. This is where the contrast issue lies. This can be solved by adding a text-shadow to this class in high contrast themes:
```css
{
text-shadow:
     -1px -1px 0 #000,  
     1px -1px 0 #000,
     -1px 1px 0 #000,
     1px 1px 0 #000;
}
```
Ref: https://css-tricks.com/adding-stroke-to-web-text/#simulation
